packages/api/src/command/medical/document/document-query.ts

```diff
@@ -80,17 +72,13 @@ export async function queryDocumentsAcrossHIEs({
     cxDocumentRequestMetadata,
   });
 
-  const cxsWithEnhancedCoverageFeatureFlagValue =
-    await getCxsWithEnhancedCoverageFeatureFlagValue();
-  if (forceQuery || !cxsWithEnhancedCoverageFeatureFlagValue.includes(patient.cxId)) {
```

Comment:
Moved to `getDocumentsFromCW()` as its CW-specific logic and it doesn't impact the DQ status for the cx anyways.